webapp/lib/controller.dart

```diff
@@ -15,15 +15,18 @@ final SYSTEM_METRICS_ROOT_COLLECTION_KEY = 'system_metrics';
 final SYSTEM_METRICS_MACHINE_NAME = 'miranda';
 final DIR_SIZE_METRICS_ROOT_COLLECTION_KEY = 'dir_size_metrics';
 
-final PROJECTS = ['Lark_KK-Project-2020-COVID19', 'Lark_KK-Project-2020-COVID19-KE-URBAN', 
+final PROJECTS = ['Lark_KK-Project-2020-COVID19', 'Lark_KK-Project-2020-COVID19-KE-URBAN',
```

Comment:
As discussed, let's work out how to read these from firebase - that may actually require a data change because of the 'documents with only sub collections and no properties don't exist' behaviour